script.js

```diff
@@ -63,4 +63,99 @@ function loadFormData() {
+    skills: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    image: 'assets/ProjectImg/modalProject.png',
+    live_version: '#',
+    link_source: '#',
+  },
+  {
+    name: 'Multi-Post Stories',
+    cardDetail: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    skills: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    image: 'assets/ProjectImg/modalProject.png',
+    live_version: '#',
+    link_source: '#',
+  },
+  {
+    name: 'Multi-Post Stories',
+    cardDetail: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    skills: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    image: 'assets/ProjectImg/modalProject.png',
+    live_version: '#',
+    link_source: '#',
+  },
+  {
+    name: 'Multi-Post Stories',
+    cardDetail: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    skills: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    image: 'assets/ProjectImg/modalProject.png',
+    live_version: '#',
+    link_source: '#',
+  },
+  {
+    name: 'Multi-Post Stories',
+    cardDetail: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    skills: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    image: 'assets/ProjectImg/modalProject.png',
+    live_version: '#',
+    link_source: '#',
+  },
+  {
+    name: 'Multi-Post Stories',
+    cardDetail: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    skills: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    image: 'assets/ProjectImg/modalProject.png',
+    live_version: '#',
+    link_source: '#',
+  },
+];
+
+function openModal() {
```

Comment:
- [ ] When I refresh the browser, without clicking the button `see project`, a modal is displaying 🖼️ That is not a behavior we were expecting. Kindly remedy that detail 🧄